### ISSUE TITLE
Text of pictogram

### ISSUE THREAD

**Issue by Capudup** [asterics/AsTeRICS-Grid#442]
It will be nice to put **the text at the top** because all robust applications recommend putting the text at the top because when the person shows the pictogram it hides the text... ?? thank you

**Comment by klues** [comment]
Yes, I'm currently working on improving the whole grid layout system and within this, I'll also implement the possibility to show the text at the top of the pictograms. Details will be discussed in issue https://github.com/asterics/AsTeRICS-Grid/issues/168 - I'll close this issue because it's a duplicate of the other one.

**Comment by Capudup** [comment]
Hello, thank you very much for your amazing work. The app is very easy to understand, even for parents. It’s great.Capucine du PelouxOrthophoniste13 avenue des frères lumière07 72 43 77 52Le 7 nov. 2024 à 11:39, Benjamin Klaus ***@***.***> a écrit :﻿
Yes, I'm currently working on improving the whole grid layout system and within this, I'll also implement the possibility to show the text at the top of the pictograms. Details will be discussed in issue #168 - I'll close this issue because it's a duplicate of the other one.
